### DISCUSSION TITLE
Button width on the Ribbon

### DISCUSSION THREAD

**Discussion by lesandrog** [Krypton-Suite/Standard-Toolkit#1818] [Q&A]
Is it possible to keep the buttons on the Ribbon the same width? Increase the width when the text is short?
![image](https://github.com/user-attachments/assets/06638dea-593a-4771-a01b-20760c3697cb)

**Comment by Smurf-IV** [comment]
Unable to check right now, But isn't there a minimum width property ?

**Comment by lesandrog** [comment]
For KryptonRibbonGroup there is a MinimumWidth property, but for KryptonRibbonGroupButton unfortunately it does not exist.
![image](https://github.com/user-attachments/assets/956cdfa8-c329-405c-805c-c745c275d969)
![image](https://github.com/user-attachments/assets/c82687d4-3ccd-43c8-afb1-37b6031ced78)

Maybe it is possible to add it in future versions?
